pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1511,30 +1464,27 @@
         [duct %pass /flog %d %flog %crud %eyre-no-channel >id=channel-id< ~]
       ::
       =/  event-id  next-id.u.channel
-      ::
-      =/  event-stream-lines=wall
-        %-  weld  :_  [""]~
-        :-  (weld "id: " (format-ud-as-integer event-id))
-        %+  turn  json-text
-        |=  =tape
-        (weld "data: " tape)
       ::  if a client is connected, send this event to them.
       ::
       =?  moves  ?=([%| *] state.u.channel)
         ^-  (list move)
+        ?~  json=(sign-to-json request-id sign)
+          moves
```

Comment:
Hmm. @eglaysher this means we might "skip" event IDs, as far as the client can tell. Considering our acking assumes but doesn't depend on contiguous numbers, and the SSE spec doesn't require IDs to be of a specific shape/order, I think this should be fine. But please do give me a second opinion!

Reply:
That probably isn't an issue.

Reply:
Ah, here's the missing piece of the puzzle: when you `%watch-as`, gall kicks the subscription if a conversion isn't available/functioning. So I think the answer here is "check if `sign-to-json` works, if it doesn't, `%kick` the subscription".

Reply:
Alright, fix or this in 5bebff3, which is in the other PR because it's at the intersection of these two changesets, and I have a conflict-averse personality. Apologies for tangling these things up so much. @eglaysher please take a look.

Reply:
(Not sure what you're asking for, but the changes in 5bebff3 look reasonable enough as error handling.)